build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
 
     dependencies {
         classpath 'gradle.plugin.gradle-plugins:jartest:1.0.1'            // necessary to depend tests on tests
-        classpath "gradle.plugin.com.github.spotbugs.snom:spotbugs-gradle-plugin:4.5.0"
+        //classpath "gradle.plugin.com.github.spotbugs.snom:spotbugs-gradle-plugin:4.5.0"
```

Comment:
Same as for the SpotBugs entry above: 
Do we need a ticket to revisit this part or can this be removed already?

Reply:
Very good idea. I created a ticket KIEKER-1854 to fix and re-enable spotbugs.